frontend/src/components/Layout/Header/SearchCourses/SearchCoursesContainer.tsx

```diff
@@ -15,12 +15,14 @@ const SearchCoursesContainer: React.FC = () => {
   useEffect(() => {
     const timer = setTimeout(() => {
       const getCourses = async (search: string) => {
-        const searchResponse = await searchAllCourses(search);
-        if (searchResponse.data) {
-          setCoursesFound(searchResponse.data);
-          setSearchOpen(true);
-        } else {
-          enqueueSnackbar('Something went wrong');
+        if (searchInputValue.length > 0) {
+          const searchResponse = await searchAllCourses(search);
+          if (searchResponse.data) {
+            setCoursesFound(searchResponse.data);
+            setSearchOpen(true);
+          } else {
+            enqueueSnackbar('Something went wrong');
```

Comment:
Here you should also add snackbar type(successSnackbar or errorSnackbar) from constants/snackbarVariant and also move the message to this file too
So you will have something like that: `enqueueSnackbar(errorSnackbarMessage.failed, errorSnackbar);`